packages/next/src/server/web/spec-extension/image-response.ts

```diff
@@ -5,54 +5,48 @@ export class ImageResponse {
       typeof import('next/dist/compiled/@vercel/og').ImageResponse
     >
   ) {
-    // @ts-expect-error - process.turbopack is a custom property
-    if (process.turbopack) {
```

Comment:
Sorry, I think we should keep the `process.turbopack` check for people still using `next-dev` (until we deprecate it), and instead check `process.turbopack && !process.env.EXERPIMENTAL_TURBOPACK`

Reply:
Is there another issue preventing it from working with `next-dev`?